draft-thomson-ppm-prss.md

```diff
@@ -530,22 +533,51 @@ Theorem 4 of {{?GKWY20}} shows that advantage comprises two components:
   of uses of the PRF (`p`), with the entropy of the pseudorandom function (`b`)
   acting to counteract attacker advantage
 
-This analysis will focus on the second component, as the first component is
-unaffected by usage of the PRF.
-
-The first component applies to an idealized PRF.  This provides statistical
-security of 2<sup>-64</sup> for AES-128 based on the birthday bound.
-
-Consequently, there is no significant benefit to limiting the second component
-to be significantly smaller than 2<sup>-64</sup>.  This analysis assumes the
-same limit for AES-256 for simplicity.  AES uses the same block size for all
-variants, so the entropy (`b`) is the same at 128 bits.
-
-For an overall advantage of 2<sup>-63</sup>, the usage component is limited to
-2<sup>-64</sup>, leading to a limit of 2<sup>64</sup> uses of the PRF, as shown
-in {{table-prf}}.
-
-This assumes that AES is modeled as an ideal pseudorandom permutation.
+The number of uses (`p`) are only affected by the second component, as the first
+component is unaffected by usage of the permutation.  However, the first
+component guides our assumptions about the number of times the attacker might be
+willing to invoke the permutation (`q`). The result shows that statistical security is
+provided based on the birthday bound. For instance, `q` being 2<sup>-64</sup>
+results in a disastrous advantage of 0.5 for the AES-128 key size of 128 bits.
+This first term therefore places an upper bound on the value that `q` can take
+before an attacker can rely solely on this computational limit.
+
+<!--
+2^{-a}/2 >= q^2/2^k/2
+2^{-a} >= q^2/2^k
+2^{k-a} >= q^2
+2^{(k-a)/2} >= q
+
+With this value of q, the second component then becomes:
+
+2^{-a}/2 >= 2pq/2^b
+    >= 2p.2^{(k-a)/2}/2^b
+    >= 2p.2^{-a/2}.2^{k/2-b}
+2^{-a/2}/4/2^{k/2-b} >= p
+2^{b-k/2-a/2-2} >= p
+p <= 2^{b-(k+a)/2-2}
```

Comment:
have you build a good intuition around `ρ` value from that paper? I read it a few times but I don't seem to understand the role of pseudo-random function in AES encryption

Reply:
The way to think of AES is as a PRP (which is a narrower formulation than a PRF).  The $\rho$ parameter is defined as the entropy of $\mathcal{R}$.

![image](https://github.com/private-attribution/i-d/assets/67641/521ac7da-15f1-42fa-8fa4-cca7541703ba)

Follow this through to Theorem 4, where they look at their $\mathrm{MMO}^\pi$ function:

![image](https://github.com/private-attribution/i-d/assets/67641/451b7494-397d-4cd5-8a5a-ad02e63572f5)

There, they use $\mathcal{R}$ in place of $\pi(\cdot)$ (i.e., AES) and add:

> In the real world, the second oracle is $\mathcal{O}^\mathrm{cr}_R(\cdot) = \mathrm{MMO}^\pi(R\oplus\cdot)$ (for $R$ sampled from a distribution $\mathcal{R}$).

From this, I conclude that $\rho$ is the entropy of AES, which is the block size of 128 bits.